cmd/kubeadm/app/cmd/util/cmdutil_test.go

```diff
@@ -70,3 +71,29 @@ func TestValidateExactArgNumber(t *testing.T) {
 		})
 	}
 }
+
+func TestGetKubeConfigPath(t *testing.T) {
+	var tests = []struct {
+		name     string
+		file     string
+		expected string
+	}{
+		{
+			name:     "provide an empty value",
+			file:     "",
+			expected: clientcmd.NewDefaultClientConfigLoadingRules().GetDefaultFilename(),
+		},
+		{
+			name:     "provide a non-empty value",
+			file:     "kubelet.kubeconfig",
+			expected: "kubelet.kubeconfig",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if GetKubeConfigPath(tt.file) != tt.expected {
+				t.Error("unexpected result")
```

Comment:
use Errorf and include "expected x, got y"

Reply:
OK, i will modify it, thanks.